pantheon/src/test/java/tech/pegasys/pantheon/RunnerTest.java

```diff
@@ -80,6 +81,7 @@ public void fullSyncFromGenesis() throws Exception {
   }
 
   @Test
+  @Ignore("Fast sync implementation in progress.")
```

Comment:
Note that NC-2178 has been filed to re-enable this once the configuration options are fully exposed (currently it fails because it's waiting for 5 peers and the test times out before fast sync does).